test/raw_tdx_quotes/README.md

```diff
@@ -0,0 +1,14 @@
+# Raw TDX Quotes (For Testing)
+
+This directory contains TDX quote and serialized Output values that we use for testing
+
+## File Structure
+
+Each directory within `raw_tdx_quotes/` is named by the 64-byte hex uncompressed public key that was used for the `TD10ReportBody.reportData` field. Each of these public key directories contains a 
+
+- `quote.bin`: which is a TDX quote binary file whose `TD10ReportBody.reportData` is the public key named by its parent directory
+- `output.bin`: which is the binary output returned by `AutomataDcapAttestationFee.verifyAndAttestOnChain` when called on the Ethereum Sepolia network
```

Comment:
ideally we'll add the key pair as well such that we can test isAllowed calls as well

Reply:
by "key pair" you mean private and public?

Forge gives us a way to fake having the private key if you already have the address, so unless we need it for some other reason, we don't need the private key. And the public key is the name of the directory under `raw_tdx_quotes/`. Let me write some tests and if it's annoying to extract the public key from the directory name, I'll add a new json file with test case config data (like the public key) in it.

Reply:
nah, i was thinking of the private key. wasn't aware of this forge feature. can be left as it is